connect/client.go

```diff
@@ -135,8 +136,8 @@ func (rs *restClient) GetVaults() ([]onepassword.Vault, error) {
 
 // GetVaults Get a list of all available vaults
 func (rs *restClient) GetVault(uuid string) (*onepassword.Vault, error) {
-	if uuid == "" {
-		return nil, errors.New("no uuid provided")
+	if !isValidUUID(uuid) {
+		return nil, errors.New("malformed vault uuid provided")
```

Comment:
These errors are duplicated a lot. If we assign them to exported variables, that has two advantages:
- The code becomes cleaner and easier to visually verify.
- Users can check which error is returned.

Reply:
Thanks for the suggestion! Just put it in code, let me know if this is what you had in mind.

Reply:
That's better indeed! 👍  Not necessarily a problem and something that can be changes easily later: with the variables unexported, the user of the package cannot use `errors.Is()` on them.